### ISSUE TITLE
start_tmux_selection() expects default key bindings

### ISSUE THREAD

**Issue by bchretien** [tmux-plugins/tmux-yank#22]
[Here](https://github.com/tmux-plugins/tmux-yank/blob/master/scripts/copy_line.sh#L36), we have:

``` sh
start_tmux_selection() {
    if [ "$TMUX_COPY_MODE" == "vi" ]; then
        # vi copy mode
        tmux send-key 'Space'
    else
        # emacs copy mode
        tmux send-key 'C-Space'
    fi
}
```

But if the user bound copy-mode `begin-selection` to another key (e.g. <kbd>V</kbd> for something more Vim-like) and remapped <kbd>Space</kbd>:

``` sh
bind -t vi-copy 'v' begin-selection
bind -t vi-copy 'Space' halfpage-down
```

 then the command-line <kbd>Prefix</kbd><kbd>Y</kbd> feature does not work.

**Comment by Alechan** [comment]
Hi, I had the same problem. When I tried to copy the current command (prefix+y) the following happened:
* Some weird characters appeared at the end of the command:
...before:  ```ls -a```
...after:  ```ls -a^A^E ```

* A "tmux notification" appeared telling me that the line was yanked succesfully to clipboard.
* The line was not yanked to clipboard.

I was using v to select so I tried having both v and space as "copy-selectors":
```.tmux
#...
#Vi like in copy mode
unbind [ # free the weird binding
set-window-option -g mode-keys vi # switch to vi bindings
bind Escape copy-mode # ctrl-a ESC put your pane in a kind of vi normal mode
bind-key -t vi-copy v begin-selection # start a visual select mode with v like in vim
bind-key -t vi-copy Space begin-selection # start a visual select mode with space like in default tmux
bind-key -t vi-copy 'C-v' rectangle-toggle # Begin rectangle-selection in copy mode (ALWAYS FOLLOW WITH A SPACE)
bind-key -t vi-copy y copy-selection # yank with y
unbind p # free another weird binding
bind p paste-buffer # paste with crtl-a p
#...
```

That's my .tmux-conf after the mods. It doesn't work I don't know why. 

PS: yanking in copy-mode works perfectly. I can yank directly to clipboard by pressing y.
PS2: I haven't tried the tmux-copycat plugin to see if this issue is also present in that plugin.
PS3: I haven't tried **bchretien** fork yet
PS4: sometimes it's easier to use the mouse but when yanking with mouse selection it's not copied to clipboard (only to the tmux buffer)

**Comment by Alechan** [comment]
Sorry, forgot my setup:
OS: Ubuntu 14.04
Term: rxvt
Term-multiplexer: tmux
shell: zsh.

**Comment by docwhat** [comment]
Can't you use the tmux 2.4 `send-key -X begin-selection` feature now?

**Comment by docwhat** [comment]
So `tmux-yank` has support for tmux 2.4; tmux 2.4 no longer has this problem.

Closing as fixed (with tmux 2.4+)
